docs/source/features/peer.rst

```diff
@@ -0,0 +1,38 @@
+.. _rst_gdeploypeer:
+
+Peer
+^^^^
+
+*peer* module allows user to add/probe remote machines to a Trusted Storage Pool,
+detach/remove a remote machine from a Trusted Storage Pool.
+action variable can be any of *probe*, *detach*, or *ignore*.
+
```

Comment:
@sac apart from ignore_peer_errors variable?
there is an 'ignore' option too in the peer.json file.
Can you please look into the same. thanks.